oozie-el-extensions/src/test/java/org/apache/oozie/extensions/TestOozieELExtensions.java

```diff
@@ -168,6 +168,8 @@ public void testDataInOptional(String expuris, String partition, String doneFlag
                 "*/US", "_DONE", },
             // With availability flag. All instances missing
             {"hdfs://localhost:8020/projects/falcon/staging/EMPTY_DIR_DONT_DELETE", "null", "_FINISH"},
+            // With availability flag and partitions. All instances missing
+            {"hdfs://localhost:8020/projects/falcon/staging/EMPTY_DIR_DONT_DELETE", "*", "_FINISH"},
```

Comment:
Code path is the same, whether there is one partition or multi-level. Whether I use * or */*/* in UT, the result will be same.